build/configs/artik053/minimal/defconfig

```diff
@@ -768,6 +767,12 @@ CONFIG_EXAMPLES_HELLO=y
 # CONFIG_SYSTEM_FOTA_HAL is not set
 # CONFIG_SYSTEM_I2CTOOL is not set
 # CONFIG_SYSTEM_INIFILE is not set
+CONFIG_SYSTEM_PREAPP_INIT=y
```

Comment:
This is not a real minimal config. build/configs/sidk_s5jt200/hello config does not include preapp_init config.